src/FluentResults/Reasons/Error.cs

```diff
@@ -24,7 +24,7 @@ public class Error : IError
         /// Get the reasons of an error
         /// </summary>
         public List<IError> Reasons { get; }
-        
+
         protected Error()
```

Comment:
There is a constructor with

```
public Error(string message, Error causedBy)
            : this(message)
```

Please change it also to an interface

```
public Error(string message, IError causedBy)
            : this(message)
```

Reply:
Done.